tools/check-cla

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+author=${1/\/cloud-init/}
+if [ `grep -c ${author} tools/.lp-to-git-user` = 0 ]; then
+    cat <<EOF
+______________________________________________________________
+
+Hello $author,
+
+Thank you for your contribution to cloud-init.
+
+In order to accept this pull request. Please confirm
+that the Contributor License Agreement (CLA) has been signed
```

Comment:
The CLA has certainly been signed, we specifically need it signed by this person:

```suggestion
that you have signed the CLA
```

(also dropped spelling out CLA as my suggestion for the previous line now includes it)